app/views/vocab_sheets/_vocab_sheet_item.html.haml

```diff
@@ -1,7 +1,7 @@
-%li.vocab_sheet_item.vocab_sheet_page_item.shadow.rounded{ style: "width:#{vocab_sheet_item_width}px;height:#{vocab_sheet_item_height}" }
+%li.vocab_sheet_item.vocab_sheet_page_item.shadow.rounded
   = hidden_field_tag "item[#{vocab_sheet_item.id}][id]", vocab_sheet_item.id, class: 'item_id'
   = link_to sign_url(vocab_sheet_item.sign_id), { class: 'drawing vocab_sheet_drawing', style: "line-height:#{vocab_sheet_image_height}px;height:#{vocab_sheet_image_height}px" } do
-    = image_tag "signs/#{vocab_sheet_image_width}-#{vocab_sheet_image_height}/#{vocab_sheet_item.drawing}"
+    = image_tag "signs/#{vocab_sheet_image_width}-#{vocab_sheet_image_height}/#{vocab_sheet_item.drawing}", style: "max-width:#{vocab_sheet_image_width - 7}px"
```

Comment:
is it possible to avoid inline styling?

Reply:
seconded, this is undesirable

Reply:
I think the reason all the inline styling is done here is because it uses helper methods to evaluate height and width values based on the page layouts.
I'm pretty sure I could change it to work through class assignment rather than style assignment from those helpers but it looks like it would add an extra chunk of work onto this branch.
What do you reckon?

Reply:
It has a higher connaisance as an inline style

Reply:
http://connascence.io/